src/backend/distributed/master/master_truncate.c

```diff
@@ -18,13 +18,21 @@
 #include "distributed/commands/utility_hook.h"
 #include "distributed/master_metadata_utility.h"
 #include "distributed/master_protocol.h"
+#include "distributed/multi_executor.h"
```

Comment:
As far as I can see, we don't need `#include "distributed/multi_executor.h"`, `#include "distributed/multi_physical_planner.h"` and `#include "distributed/relay_utility.h"`